anzu.el

```diff
@@ -165,7 +165,7 @@
   (list :count count :overflow overflow :positions positions))
 
 (defsubst anzu--case-fold-search (input)
-  (when case-fold-search
+  (when isearch-case-fold-search
```

Comment:
Is this due to a regression in `isearch`?   At first glance it looks like `isearch-case-fold-search` is supposed to be initialized to the value of `case-fold-search`, and I can't see why this change would be necessary.

Reply:
Ah, now I see.  This patch fixes the counter during `isearch-forward` including 
 - case insensitive (the search string is all lower case)
 - case sensitive (the search string has at least some upper case characters)
 - case sensitive (the user has typed `M-c` during isearch)

However, it doesn't affect the behavior of `anzu-isearch-query-replace` after this search is done.  For example, if the user has forced a case sensitive isearch with `M-x` and then enters a search string that is entirely lower case, the subsequent `anzu-isearch-query-replace` will replace the case *in*sensitive matches for that lower case string. Let's treat query replace a separate issue after merging this.